proxy/main.go

```diff
@@ -16,7 +16,8 @@ import (
 // TODO: to be managed externally
 const ZdmVersionString = "2.2.0"
 
-var displayVersion = flag.Bool("version", false, "Display the ZDM proxy version and exit")
+var displayVersion = flag.Bool("version", false, "display the ZDM proxy version and exit")
```

Comment:
We should move all the code on this file to a separate file that always builds regardless of whether profiling mode is enabled or not. And the config file flag should be moved to that file as well.

The only difference between `main_profiling.go` and `main.go` should be the profiling stuff. At the moment `displayVersion` is not enabled during profile mode which makes no sense.

I know this wasn't done on this PR but since you're adding a new configfile flag we should probably get this fixed now.

Reply:
I did not want to add new small file and moved the code to `launch.go`. I refactored also version displaying not to duplicate code. See if it is OK for you.